tests/src/python/test_qgsexpression.py

```diff
@@ -29,10 +31,6 @@ def testfun(values, feature, parent):
         """ Function help """
         return f"Testing_{values[0]}"
 
-    @qgsfunction(args="auto", group='testing', register=False)
```

Comment:
Can we leave this? IIUC, this is still supported even if deprecated

Reply:
Actually we can't. The point of this function was to be later tested in 
```Python
def testAutoCountsCorrectArgs(self):
    function = self.autocount
    args = function.params()
    self.assertEqual(args, 3)
```

But with this PR, function.params() is always equal to -1